src/frontend/Navigation/Stack/AppScreens.tsx

```diff
@@ -441,11 +446,19 @@ export const createDefaultScreenGroup = ({
         name="AudioSavedBottomSheet"
         component={AudioSavedBottomSheet}
       />
+      <RootStack.Screen name="InviteReceived" component={InviteReceived} />
+      <RootStack.Screen
+        name="InviteSuccessfullyJoined"
+        component={InviteSuccessfullyJoined}
+      />
+      <RootStack.Screen name="InviteCancelled" component={InviteCancelled} />
```

Comment:
an absolute nit but i believe core spells it using one `l` i.e. `canceled`.

```suggestion
      <RootStack.Screen name="InviteCanceled" component={InviteCanceled} />
```

the practical argument is i'll pose is that the consistency helps for grepping through code 😄